modules/games/definitions/jesonmor/generators.ts

```diff
@@ -0,0 +1,28 @@
+// Here you define the generators (artifact creators) for your game.
+// The type analyser will look at this file to see what generators are available,
+// and also what artifact layers there are. When you add/remove a generator or
+// change the names of the layers you draw to, rerun the type analyser!
+
+import {JesonmorDefinition} from "./_types";
+
+const jesonmorGenerators: JesonmorDefinition["generators"] = {
+  findmovetargets: {
+    start: "selectunit",
+    type: "neighbour",
+    dirs: "knight",
+    draw: {
+      neighbours: {
+        unlessover: "myunits",
+        condition: {
+          or: [
+            {not: {overlaps: ["oppunits", "center"]}},
+            {anyat: ["center", ["target"]]}
+          ]
+        }
```

Comment:
Caused by typo in my earlier suggestion 😊 

```suggestion
        },
```